docs/reference/modules/uploadfs.md

```diff
@@ -29,14 +29,48 @@ module.exports = {
+The options documentation for this module are split into those that are general storage options and those that are service-specific for local, AWS S3, Microsoft Azure, and Google Cloud Storage. A number of the options are described in greater detail in the official documentation for the specific service and links are provided. If you are using an S3 compatible service, the options provided through the `storage: 's3'` option are a good starting point and may be sufficient, along with the `endpoint` option which is needed to point to an alternative provider.
 
 ## General Storage Options
 |  Property | Type | Description |
 |---|---|---|
 | [`cdn`](#cdn) | String \|\| Object | Takes either a string representing the URL of the CDN, or an object with `url` and `enabled` properties. |
 | `prefix` | String | Allows prefixing of static assets that share a single, multisite backend bucket |
 | [`storage`](#storage) | String \|\| Object  | Storage backend specified as string for built-in solutions or an object for custom storage. |
+| [`tempPath`](#temppath) | String | Defaults to `data/temp/uploadfs` relative to the project root. Directory location for creating the temporary folder and files during processing. |
+
+### `cdn`
+If you choose to host assets on a CDN, set the `cdn` property value to the URL of the CDN. This property can also take an object with two properties - `url` and `enabled`. The `url` property value is the URL of the CDN and `enabled` takes a boolean which determines if the CDN will be used. This results in Apostrophe outputing URLs that point to the CDN rather than directly to the storage backend. Configuring your CDN of choice to automatically obtain files from your storage backend is your responsibility.
+
+#### Example
+
+<AposCodeBlock>
+
+```javascript
+module.exports = {
+  options: {
+    uploadfs: {
+      cdn: {
+        url:'https://wpc.cdnurl.com',
+        enabled: true
+      }
+    }
+  }
+};
+
+```
+  <template v-slot:caption>
+    modules/@apostrophecms/uploadfs/index.js
+  </template>
+</AposCodeBlock>
+
+### `storage` 
+This property takes either a string designating one of the built-in storage options to be used, or an object for custom storage needs. Built-in values are `azure` (Microsoft Azure), `gcs` (Google Cloud Storage), `local` (local file storage only), and `s3` (Amazon Simple Storage Service). See the [`s3.js`](https://github.com/apostrophecms/uploadfs/blob/main/lib/storage/s3.js) file for an example of creating a custom storage solution.
+
+### `tempPath`
+During processing of files, the `uploadfs` module first copies them to a temporary location until the pipeline is finished. By default this is set to `data/temp/uploadfs` by the `@apostrophecms/uploadfs` module. The `tempPath` property takes a local directory path for creating the `/temp` folder that houses these files. It is deleted after the build process.
```

Comment:
It isn't deleted after the build process. The main tempPath folder hangs around and gets used every time there is an image upload, for image processing tasks, among other uses that might indeed be during the build process.

Reply:
Ahhh - missed this because VSC folds in folders that only have folders as content. I never noticed the temp directory, so I thought it was deleted and I missed it in the code. Deleted.